data/fields/cuisine.json

```diff
@@ -33,7 +33,21 @@
             "greek": "Greek",
             "vietnamese": "Vietnamese",
             "turkish": "Turkish",
-            "lebanese": "Lebanese"
+            "lebanese": "Lebanese",
+            "indonesian": "Indonesian",
+            "malaysian": "Malaysian",
+            "taiwanese": "Taiwanese",
+            "juice": "Fruit Juice",
+            "bubble_tea": "Bubble Tea",
+            "hot_dog": "Hot Dog",
+            "cake": "Cake",
+            "pankcake": "Pancake",
+            "pasta": "Pasta",
+            "salad": "Salad",
+            "chocolate": "Chocolate",
```

Comment:
was there a specific reason to include this value? It seems to not be used too often and is also not included on the [`cuisine`](https://wiki.openstreetmap.org/wiki/Key:cuisine) page on the wiki.

Reply:
It's got 119 uses, and when I was looking through the list it's one that I could think of a number of places where their main cusine is chocolate.

Reply:
I've added to the wiki now